srcpkgs/beancount/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'beancount'
+pkgname=beancount
+version=2.1.3
+revision=1
+build_style=python3-module
+pycompile_module="beancount"
+hostmakedepends="python3-devel python3-setuptools"
```

Comment:
is the -devel in hostmakedepends really necessary?

Reply:
Seems to work without it.